core/adapters/doc.go

```diff
@@ -76,11 +76,101 @@
 // WARNING: The random adapter as implemented is not verifiable.
 // Outputs from this adapters are not verifiable onchain as a fairly-drawn random samples.
 // As a result, the oracle potentially has complete discretion to instead deliberately choose
-// values with favorable onchain outcomes. Don't use it for a lottery, for instance, unless 
+// values with favorable onchain outcomes. Don't use it for a lottery, for instance, unless
 // you fully trust the oracle not to pick its own tickets.
 // We intend to either improve it in the future, or introduce a verifiable alternative.
 // For now it is provided as an alternative to making web requests for random numbers,
 // which is similarly unverifiable and has additional possible points of failure.
 //  { "type": "Random" }
 //
+// EthTxABIEncode
+//
+// The EthTxABIEncode adapter serializes the contents of a json object as transaction data
+// calling an arbitrary function of a smart contract. See
+// https://solidity.readthedocs.io/en/v0.5.11/abi-spec.html#formal-specification-of-the-encoding
+// for the serialization format. We currently support all types that solidity contracts
+// as of solc v0.5.11 can decode, i.e. address, bool, bytes*, int*, uint*, arrays (e.g. address[2]),
```

Comment:
Might want to be more explicit about the globbing here, too.

Reply:
Thanks, fixed.